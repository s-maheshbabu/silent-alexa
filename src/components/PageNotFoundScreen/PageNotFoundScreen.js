import React from "react";
import "./PageNotFoundScreen.css";

export default function PageNotFoundScreen() {
  return (
    <div id="pagenotfound-screen">
      <h1> 404 </h1>
      <h1> This is just a placeholder. We might want to have a link here to let the user try logging in again or go home. </h1>
    </div>
  );
}
