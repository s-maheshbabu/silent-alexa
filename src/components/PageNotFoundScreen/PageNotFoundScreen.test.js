import React from "react";
import { render, cleanup } from '@testing-library/react';
import PageNotFoundScreen from "./PageNotFoundScreen";

afterEach(cleanup);

it("renders without crashing", () => {
  const { asFragment } = render(<PageNotFoundScreen />);
  expect(asFragment(<PageNotFoundScreen />)).toMatchSnapshot();
});
