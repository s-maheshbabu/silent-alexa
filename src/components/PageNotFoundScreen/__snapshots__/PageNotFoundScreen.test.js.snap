// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    id="pagenotfound-screen"
  >
    <h1>
       404 
    </h1>
    <h1>
       This is just a placeholder. We might want to have a link here to let the user try logging in again or go home. 
    </h1>
  </div>
</DocumentFragment>
`;
